fury/primitive.py

```diff
@@ -670,25 +670,150 @@ def prim_frustum():
 
     """
     vertices = np.array([[-.5, -.5, .5],
-                        [.5, -.5, .5],
-                        [.5, .5, .5],
-                        [-.5, .5, .5],
-                        [-1, -1, -.5],
-                        [1, -1, -.5],
-                        [1, 1, -.5],
-                        [-1, 1, -.5]])
+                         [.5, -.5, .5],
+                         [.5, .5, .5],
+                         [-.5, .5, .5],
+                         [-1, -1, -.5],
+                         [1, -1, -.5],
+                         [1, 1, -.5],
+                         [-1, 1, -.5]])
     triangles = np.array([[4, 6, 5],
-                         [6, 4, 7],
-                         [0, 2, 1],
-                         [2, 0, 3],
-                         [4, 3, 0],
-                         [3, 4, 7],
-                         [7, 2, 3],
-                         [2, 7, 6],
-                         [6, 1, 2],
-                         [1, 6, 5],
-                         [5, 0, 1],
-                         [0, 5, 4]], dtype='u8')
+                          [6, 4, 7],
+                          [0, 2, 1],
+                          [2, 0, 3],
+                          [4, 3, 0],
+                          [3, 4, 7],
+                          [7, 2, 3],
+                          [2, 7, 6],
+                          [6, 1, 2],
+                          [1, 6, 5],
+                          [5, 0, 1],
+                          [0, 5, 4]], dtype='u8')
     vertices /= 2
     triangles = fix_winding_order(vertices, triangles, clockwise=True)
     return vertices, triangles
+
+
+def prim_cylinder(radius=1, height=3, sectors=36, capped=True):
+    """Return vertices and triangle for a cylinder.
+
+    Parameters
+    ----------
+    radius: float
+        Radius of the cylinder
+    height: float
+        Height of the cylinder
+    sectors: int
+        Sectors in the cylinder
+    capped: bool
+        Whether the cylinder is capped at both ends or open
+
+    Returns
+    -------
+    vertices: ndarray
+        vertices coords that compose our cylinder
+    triangles: ndarray
+        triangles that compose our cylinder
+    """
+
+    if not isinstance(sectors, int):
```

Comment:
check also if sectors are meaningful:  

`if not isinstance(sectors, int) and sectors < 8:`